mscore/inspector/inspector.cpp

```diff
@@ -160,13 +160,10 @@ void Inspector::setElements(const QList<Element*>& l)
 
             bool sameTypes = true;
             for (Element* ee : _el) {
-                  if (_element->type() != ee->type())
+                  if ((_element->type() != ee->type()) && // different and
+                      (!_element->isSlurTieSegment() && !ee->isSlurTieSegment()) || // neither Slur nor Tie either side, or
+                      (ee->isNote() && toNote(ee)->chord()->isGrace() != toNote(_element)->chord()->isGrace())) // HACK
```

Comment:
To be honest: I don't quite understand that that HACK condition is all about, why should notes and grace notes not be treated as same types?